usecase/usecase_test.go

```diff
@@ -1 +1,150 @@
 package usecase
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/rayl17/academy-go-q42021/mocks"
+	"github.com/rayl17/academy-go-q42021/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPokemonUsecase_GetPokemonByName(t *testing.T) {
+	tests := []struct {
+		name        string
+		pokemonName string
+		want        *model.Pokemon
+		expectedErr error
+	}{
+		{
+			name:        "prueba",
```

Comment:
The name of this scenario is not descriptive. Should describe the scenario you are testing.